src/Plugin.php

```diff
@@ -78,4 +81,22 @@ public function wcBootstrap(Site $site, SingleSwitcher $siteSwitcher)
         add_action('woocommerce_new_product', [$wooCommerceGallery, 'saveGalleryIds']);
         add_action('woocommerce_update_product', [$wooCommerceGallery, 'saveGalleryIds']);
     }
+
+    /**
+     * Integration for ACF, specifically Image fields
+     *
+     * @param Site $site
+     * @param SingleSwitcher $siteSwitcher
+     */
+    public function acfBootstrap(Site $site, SingleSwitcher $siteSwitcher)
+    {
+        // ACF can be included within a theme too - check in after_setup_theme action
+        // https://www.advancedcustomfields.com/resources/including-acf-within-a-plugin-or-theme/
+        add_action('after_setup_theme', function () use ($site, $siteSwitcher) {
+            if (\function_exists('get_field')) {
+                $acfImage = new ACF\Image($site, $siteSwitcher);
+                add_filter('acf/load_value/type=image', array($acfImage, 'acfLoadValue'), 10, 3);
+            }
+        });
```

Comment:
```suggestion
        \add_action('after_setup_theme', function () use ($site, $siteSwitcher) {
            if (!\function_exists('get_field')) {
                return;
            }
            
            $acfImage = new ACF\Image($site, $siteSwitcher);
            \add_filter('acf/load_value/type=image', [$acfImage, 'acfLoadValue'], 10, 3);
        });
```

That will make more simple to introduce more code if needed. Using the precondition to return early make more clear what's the case on when the action must not be performed.

Reply:
You are right - this makes it more clear.
The code is now refactored into `\MultisiteGlobalMedia\ACF\Image::afterSetupTheme`